db/schema.rb

```diff
@@ -48,6 +48,7 @@
   create_table "parties", force: :cascade do |t|
     t.string "name"
     t.string "color"
+    t.string "short_name"
```

Comment:
I don't think we need this as per Zulip discussion; instead let's define a `#short_name` method on the model to do the lookup against `REFERENCE_DATA`.